#! /bin/sh /usr/share/dpatch/dpatch-run
## 30Fix_tracker_backend.dpatch by Michael Biebl <biebl@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix compatibility with Tracker 0.8.

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' catfish-0.3.2~/catfish.py catfish-0.3.2/catfish.py
--- catfish-0.3.2~/catfish.py	2010-04-13 02:01:04.000000000 +0200
+++ catfish-0.3.2/catfish.py	2010-04-13 02:01:17.249556252 +0200
@@ -629,6 +629,7 @@
                     if self.options.debug: print 'Debug:', msg
                     query = generic_query()
                 for filename in query.run(keywords, folder, exact, hidden, limit):
+                    filename = filename.strip()
                     if self.abort_find or len(listmodel) == limit: break
                     filename = filename.split(os.linesep)[0]
                     # Convert uris to filenames
